### PR TITLE
BE-8028: Each helper

### PR DESCRIPTION
**Pull request by ajanata**
Run a subtest for each of the given values. Should help with table based testing.

### CODE REVIEW

**Review by jeffatgametime — APPROVED** [review]
importante

**Review by waitstime — APPROVED** [review]

File: helper.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package testy
+
+import (
+	"fmt"
+)
+
+// TestEach runs tester as a subtest for each value in values. The values should have a good default string
+// representation so the subtest names are legible. (Consider implementing fmt.Stringer for complex structs.)
+func TestEach[V any](t TestingT, values []V, tester func(TestingT, V)) {
```
Comment (ajanata) [inline helper.go:9]: actually I think `TestEach[V any](t TestingT, tester (func TestingT, v), values ...V)` might make more sense. Thoughts?

Comment (ajanata) [reply helper.go:9]: actually no, that makes writing the func inline really ugly

Comment (jeffatgametime) [reply helper.go:9]: oh yeah that would be ugly

**Pull request merged**